cvat-ui/src/components/annotation-page/top-bar/filters-modal.tsx

```diff
@@ -0,0 +1,266 @@
+// Copyright (C) 2021 Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+
+import { DownOutlined } from '@ant-design/icons';
+import { changeAnnotationsFilters, fetchAnnotationsAsync, showFilters } from 'actions/annotation-actions';
+import { Dropdown, Menu } from 'antd';
+import Button from 'antd/lib/button';
+import Modal from 'antd/lib/modal';
+import React, { useEffect, useState } from 'react';
+import {
+    Builder, Config, ImmutableTree, JsonLogicTree, Query, Utils as QbUtils,
+} from 'react-awesome-query-builder';
+import AntdConfig from 'react-awesome-query-builder/lib/config/antd';
+import 'react-awesome-query-builder/lib/css/styles.css';
+import { useDispatch, useSelector } from 'react-redux';
+import { CombinedState } from 'reducers/interfaces';
```

Comment:
Can you organize imports, please? For example react import should be at the top of the file. For ant components you can use separate imports like `ant/lib/<component>` for bundle minification. Local import should be after all external components import.

Reply:
OK